### ISSUE TITLE
Disconnect bei unfollow Messaging

### ISSUE THREAD

**Issue by WoweePaw** [YannickFricke/TwitchUnfollowNotifier#4]
Wenn der Bot zu viele Nachrichten zu schnell raushaut scheint dieser automatisch von Twitch getrennt zu werden und kann anschließend nicht neu verbinden. :)

**Comment by YannickFricke** [comment]
@D3thw0lf Gibt es dazu ein Log?

**Comment by WoweePaw** [comment]
Nope. Er sagt nur er versucht es erneut bis halt das Limit reached ist und dann beendet es sich. :3
